torch/utils/dlpack.py

```diff
@@ -18,6 +18,7 @@ class DLDeviceType(enum.IntEnum):
     kDLVPI = 9,
     kDLROCM = 10,
     kDLExtDev = 12,
+    kDLOneAPI = 14,
```

Comment:
Did you leave out `15` and `16` on purpose here because they're not expected to have a Python library available?

Reply:
I am just not sure whether I should add `15` and `16`.
Do you think it is required for better user's API?

Reply:
I think it's fine to leave out until there's case to use the Python API on these devices (which I'm not sure exists).